cmd/metrics-server/app/options/options.go

```diff
@@ -57,6 +57,9 @@ func (o *Options) Validate() []error {
 	if o.MetricResolution < 10*time.Second {
 		errors = append(errors, fmt.Errorf("metric-resolution should be a time duration at least 10s, but value %v provided", o.MetricResolution))
 	}
+	if o.MetricResolution < o.KubeletClient.KubeletRequestTimeout {
```

Comment:
Can we use previous setting of 90% as max timeout? `o.MetricResolution < o.KubeletClient.KubeletRequestTimeout * 9 / 10`

Reply:
I have changed it to be consistent with the original

Reply:
Hi, @serathius , I think it should be  `o.MetricResolution * 9 / 10 < o.KubeletClient.KubeletRequestTimeout ` ?
Error when    `o.MetricResolution * 9 / 10 < o.KubeletClient.KubeletRequestTimeout `.